docs/DeveloperGuide.adoc

```diff
@@ -767,16 +807,139 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+=== Use case: Search for person
+
+*MSS*
+
+1.  User requests to search for a person with criteria
+2.  AddressBook shows a list of persons who match the criteria
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
+
+* 2a. No such person was found.
++
```

Comment:
what's the difference between these two extensions? By the way, they seem to have the same number `2a`.  The 2nd one should be `2b`.

Reply:
The first case is when the list is empty, thus the search cannot even begin, while the second case refers to when the search completes, but the targeted person cannot be found. 
I was mistaken about the extension numbering! I will take note of this in the future.